### PR TITLE
[docs] "What's new in Svelte" January newsletter 

### PR DESCRIPTION
**Pull request by DreaminDani**
Let me know if there's anything I missed!

### CODE REVIEW

File: sites/svelte.dev/content/blog/2023-01-01-whats-new-in-svelte-january-2023.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+author: Daniel Sandoval
+authorURL: https://desandoval.net
+---
+
+It's been just two weeks since the release of [SvelteKit 1.0](https://svelte.dev/blog/announcing-sveltekit-1.0)! If you haven't yet, check out the [livestream](https://www.youtube.com/watch?v=N4BRVkQVoMc), [new website](https://kit.svelte.dev/) and [learn.svelte.dev](https://learn.svelte.dev/) to learn all the features of SvelteKit step-by-step.
+
+Or, if you prefer, just [subscribe to this newsletter](https://svelte.substack.com/) 😉 Let's dive into the details...
+
+## What's new in SvelteKit
+- @sveltejs/kit 1.0 is out! Starting from now all releases follow semver and changes will be listed as Major/Minor/Patch ([CHANGELOG](https://github.com/sveltejs/kit/blob/master/packages/kit/CHANGELOG.md#100))
```
Comment (benmccann) [inline sites/svelte.dev/content/blog/2023-01-01-whats-new-in-svelte-january-2023.md:13]:
```suggestion
- `@sveltejs/kit` 1.0 is out! All future releases will follow semver and changes will be listed as major/minor/patch in the [CHANGELOG](https://github.com/sveltejs/kit/blob/master/packages/kit/CHANGELOG.md#100).
```

File: sites/svelte.dev/content/blog/2023-01-01-whats-new-in-svelte-january-2023.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+- The Vite plugin is now split in two - replacing the need for an unnecessary middleware plugin ([#7990](https://github.com/sveltejs/kit/pull/7990))
```
Comment (benmccann) [inline sites/svelte.dev/content/blog/2023-01-01-whats-new-in-svelte-january-2023.md:14]:
```suggestion
- Improved support for Storybook and Histoire ([#7990](https://github.com/sveltejs/kit/pull/7990)). Work is ongoing to fully support those tools ([storybook#20239](https://github.com/storybookjs/storybook/pull/20239)).
```

File: sites/svelte.dev/content/blog/2023-01-01-whats-new-in-svelte-january-2023.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+- Kit's custom `svelte-preprocess` has been replaced by `vitePreprocess` ([#8036](https://github.com/sveltejs/kit/pull/8036))
```
Comment (benmccann) [inline sites/svelte.dev/content/blog/2023-01-01-whats-new-in-svelte-january-2023.md:15]:
```suggestion
- `vitePreprocess` is now the default preprocessor. Please see [the docs](https://kit.svelte.dev/docs/integrations#preprocessors) for differences between `vitePreprocess` and `svelte-preprocess` ([#8036](https://github.com/sveltejs/kit/pull/8036)).
```

File: sites/svelte.dev/content/blog/2023-01-01-whats-new-in-svelte-january-2023.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
+---
+title: "What's new in Svelte: January 2022"
+description: "SvelteKit 1.0 release, directions in custom transitions and new HTML/Svelte typings"
```
Comment (benmccann) [inline sites/svelte.dev/content/blog/2023-01-01-whats-new-in-svelte-january-2023.md:3]: directions in custom transitions doesn't feel as big as the other two to me. I don't think we have to have three
```suggestion
description: "SvelteKit 1.0 release and type definitions for Svelte elements."
```

File: sites/svelte.dev/content/blog/2023-01-01-whats-new-in-svelte-january-2023.md [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+- `invalid()` is now `fail()` and `ValidationError` is now `ActionFailure` ([#8012](https://github.com/sveltejs/kit/pull/8012))
+- Kit will now throw an error on invalid load response ([#8003](https://github.com/sveltejs/kit/pull/8003))
+- SvelteKit is now using Vite 4 and requires a Svelte peerDependency of ^3.54.0 ([#7543](https://github.com/sveltejs/kit/pull/7543))
+- Shells are now prerendered when `ssr` is false and `prerender` is not false - ensure prerender is false when ssr is also false ([#8131](https://github.com/sveltejs/kit/pull/8131))
+- Warnings and errors about removed/changed APIs have been removed ([#8019](https://github.com/sveltejs/kit/pull/8019))
+
+## What's new in Svelte
+- The `options.direction` argument can now be passed to custom transition functions (**3.54.0**, [#3918](https://github.com/sveltejs/svelte/issues/3918))
+- Variables can now be updated from a `@const` declared function (**3.54.0**, [#7843](https://github.com/sveltejs/svelte/issues/7843))
+- `svelte/elements` has been added for HTML/Svelte typings (**3.55.0**, [#7649](https://github.com/sveltejs/svelte/pull/7649))
```
Comment (benmccann) [inline sites/svelte.dev/content/blog/2023-01-01-whats-new-in-svelte-january-2023.md:32]:
```suggestion
- `svelte/elements` has been added for Svelte/HTML type definitions (**3.55.0**, [#7649](https://github.com/sveltejs/svelte/pull/7649))
```

File: sites/svelte.dev/content/blog/2023-01-01-whats-new-in-svelte-january-2023.md (hunk 4 above)
Comment (benmccann) [inline sites/svelte.dev/content/blog/2023-01-01-whats-new-in-svelte-january-2023.md:3]:
```suggestion
description: "SvelteKit 1.0, learn.svelte.dev, and type definitions for Svelte elements."
```

File: sites/svelte.dev/content/blog/2023-01-01-whats-new-in-svelte-january-2023.md [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+
+Or, if you prefer, just [subscribe to this newsletter](https://svelte.substack.com/) 😉 Let's dive into the details...
+
+## What's new in SvelteKit
+- @sveltejs/kit 1.0 is out! Starting from now all releases follow semver and changes will be listed as Major/Minor/Patch ([CHANGELOG](https://github.com/sveltejs/kit/blob/master/packages/kit/CHANGELOG.md#100))
+- The Vite plugin is now split in two - replacing the need for an unnecessary middleware plugin ([#7990](https://github.com/sveltejs/kit/pull/7990))
+- Kit's custom `svelte-preprocess` has been replaced by `vitePreprocess` ([#8036](https://github.com/sveltejs/kit/pull/8036))
+
+**Breaking changes:**
+- Unknown exports (except when starting with an underscore) are no longer allowed from +(layout|page)(.server)?.js and +server.js files ([#7878](https://github.com/sveltejs/kit/pull/7878))
```
Comment (benmccann) [inline sites/svelte.dev/content/blog/2023-01-01-whats-new-in-svelte-january-2023.md:18]:
```suggestion
- Unknown exports (except when starting with an underscore) are no longer allowed from `+(layout|page)(.server)?.js` and `+server.js` files ([#7878](https://github.com/sveltejs/kit/pull/7878))
```

File: sites/svelte.dev/content/blog/2023-01-01-whats-new-in-svelte-january-2023.md [hunk 7]
```diff
@@ -0,0 +1,101 @@
...
+- `__data.json` is now stripped from url ([#7979](https://github.com/sveltejs/kit/pull/7979))
```
Comment (benmccann) [inline sites/svelte.dev/content/blog/2023-01-01-whats-new-in-svelte-january-2023.md:19]:
```suggestion
- `__data.json` is now stripped from URL ([#7979](https://github.com/sveltejs/kit/pull/7979))
```

File: sites/svelte.dev/content/blog/2023-01-01-whats-new-in-svelte-january-2023.md [hunk 8]
```diff
@@ -0,0 +1,101 @@
...
+- `sveltekit()` will now return a promise of array of Vite plugins ([#7994](https://github.com/sveltejs/kit/pull/7994))
```
Comment (benmccann) [inline sites/svelte.dev/content/blog/2023-01-01-whats-new-in-svelte-january-2023.md:20]:
```suggestion
- `sveltekit()` will now return a promise for an array of Vite plugins ([#7994](https://github.com/sveltejs/kit/pull/7994))
```

File: sites/svelte.dev/content/blog/2023-01-01-whats-new-in-svelte-january-2023.md [hunk 9]
```diff
@@ -0,0 +1,101 @@
...
+- A new `embedded` option, turned off by default, helps with link clicks when embedding SvelteKit ([docs](https://kit.svelte.dev/docs/configuration), [#7969](https://github.com/sveltejs/kit/pull/7969))
+- Automatic fallback generation has been replaced with `builder.generateFallback(fallback)` ([#8013](https://github.com/sveltejs/kit/pull/8013))
+- `invalid()` is now `fail()` and `ValidationError` is now `ActionFailure` ([#8012](https://github.com/sveltejs/kit/pull/8012))
+- Kit will now throw an error on invalid load response ([#8003](https://github.com/sveltejs/kit/pull/8003))
```
Comment (benmccann) [inline sites/svelte.dev/content/blog/2023-01-01-whats-new-in-svelte-january-2023.md:24]:
```suggestion
- SvelteKit will now throw an error on invalid load response ([#8003](https://github.com/sveltejs/kit/pull/8003))
```

File: sites/svelte.dev/content/blog/2023-01-01-whats-new-in-svelte-january-2023.md [hunk 10]
```diff
@@ -0,0 +1,101 @@
...
+- SvelteKit is now using Vite 4 and requires a Svelte peerDependency of ^3.54.0 ([#7543](https://github.com/sveltejs/kit/pull/7543))
```
Comment (benmccann) [inline sites/svelte.dev/content/blog/2023-01-01-whats-new-in-svelte-january-2023.md:25]:
```suggestion
- SvelteKit is now using Vite 4 and requires a Svelte `peerDependency` of `^3.54.0` ([#7543](https://github.com/sveltejs/kit/pull/7543))
```

File: sites/svelte.dev/content/blog/2023-01-01-whats-new-in-svelte-january-2023.md [hunk 11]
```diff
@@ -0,0 +1,101 @@
...
+- TypeScript version is now 4.9
+- Svelte version is now 3.55
+
+The following features have also been released:
+- missing handler quick fix ([#1731](https://github.com/sveltejs/language-tools/pull/1731))
+- add Svelte anchor missing attribute code action ([#1730](https://github.com/sveltejs/language-tools/pull/1730))
+- better commit characters handling ([#1742](https://github.com/sveltejs/language-tools/pull/1742))
+- add `--preserveWatchOutput` option ([#1715](https://github.com/sveltejs/language-tools/pull/1715))
+- enhance Quickfixes to include Svelte Stores ([#1789](https://github.com/sveltejs/language-tools/pull/1789))
+- only show SvelteKit files context menu in kit project ([#1771](https://github.com/sveltejs/language-tools/pull/1771))
```
Comment (benmccann) [inline sites/svelte.dev/content/blog/2023-01-01-whats-new-in-svelte-january-2023.md:46]:
```suggestion
- only show SvelteKit files context menu in SvelteKit projects ([#1771](https://github.com/sveltejs/language-tools/pull/1771))
```

File: sites/svelte.dev/content/blog/2023-01-01-whats-new-in-svelte-january-2023.md [hunk 12]
```diff
@@ -0,0 +1,101 @@
...
+- use satisfies operator if possible ([#1770](https://github.com/sveltejs/language-tools/pull/1770))
```
Comment (benmccann) [inline sites/svelte.dev/content/blog/2023-01-01-whats-new-in-svelte-january-2023.md:47]:
```suggestion
- use the `satisfies` operator if possible ([#1770](https://github.com/sveltejs/language-tools/pull/1770))
```

File: sites/svelte.dev/content/blog/2023-01-01-whats-new-in-svelte-january-2023.md [hunk 13]
```diff
@@ -0,0 +1,101 @@
...
+_To Read_
+- [Rendering emails with Svelte](https://escape.tech/blog/sveltemails/) by Gautier Ben Aïm
+- [Now That React is Dead, What’s the Next Big Thing?](https://javascript.plainenglish.io/now-that-react-js-is-dead-whats-the-next-big-thing-7fa72a36a69b) by Somnath Singh
+- [What is SvelteKit? And Why Should You Care?](https://blog.tiia.rocks/what-is-sveltekit-and-why-should-you-care) by Tila
+- [Sveltekit API endpoints](https://www.jefmeijvis.com/post/006-sveltekit-api-endpoints) by Jef Meijvis
+- [Chart.js 4.0](https://github.com/chartjs/Chart.js/discussions/10977) has been released, with updated Svelte support
+- [Creating A Custom Svelte Media Query Store](https://pqina.nl/blog/svelte-media-query-store/) by Rik Schennink
+
+**Libraries, Tools & Components**
+- [Konsta UI](https://konstaui.com/) is pixel perfect mobile UI components built with Tailwind CSS for React, Vue & Svelte
```
Comment (benmccann) [inline sites/svelte.dev/content/blog/2023-01-01-whats-new-in-svelte-january-2023.md:89]:
```suggestion
- [Konsta UI](https://konstaui.com/) is a library of pixel perfect mobile UI components built with Tailwind CSS for React, Vue & Svelte
```

**Comment by DreaminDani** [comment]
Looking much better thanks to @benmccann 's suggestions. Feel free to merge on Dec 31/Jan 1 unless there's any other edits folks would like to make before then :)

File: sites/svelte.dev/content/blog/2023-01-01-whats-new-in-svelte-january-2023.md [hunk 14]
```diff
@@ -0,0 +1,101 @@
+---
+title: "What's new in Svelte: January 2022"
+description: "SvelteKit 1.0, learn.svelte.dev, and type definitions for Svelte elements."
+author: Dani Sandoval
+authorURL: https://dreamindani.com
+---
+
+It's been just two weeks since the release of [SvelteKit 1.0](https://svelte.dev/blog/announcing-sveltekit-1.0)! If you haven't yet, check out the [livestream](https://www.youtube.com/watch?v=N4BRVkQVoMc), [new website](https://kit.svelte.dev/) and [learn.svelte.dev](https://learn.svelte.dev/) to learn all the features of SvelteKit step-by-step.
+
+Or, if you prefer, just [subscribe to this newsletter](https://svelte.substack.com/) 😉 Let's dive into the details...
```
Comment (benmccann) [inline sites/svelte.dev/content/blog/2023-01-01-whats-new-in-svelte-january-2023.md:10]: I'm not sure this sentence really meshes with the rest. Subscribing will give you future updates, but it won't tell you about the stuff that happened this month, which you can just read below. I think if we were going to have a link we'd probably want to make it be https://svelte.dev/blog/rss.xml as the people who have chosen to read on svelte.dev might prefer to read it there in the future as well.
```suggestion
Let's dive into the details...
```

Comment (DreaminDani) [reply sites/svelte.dev/content/blog/2023-01-01-whats-new-in-svelte-january-2023.md:10]: Good call. Removing

**Comment by DreaminDani** [comment]
@benmccann can we merge this, when you get a chance? Cc @dummdidumm 

Happy new year!

**Pull request merged**